inc/saiswitch.h

```diff
@@ -500,6 +500,9 @@ typedef enum _sai_switch_attr_t
        Default no map */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP,
 
+    /* Enable switch shell [bool] (default to false) */
+    SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE,
```

Comment:
this switch shell is vendor specific and is more like hw debug, maybe we can rename as SWITCH HW SHELL. Will add more into the comments.